subprojects/docs/src/docs/release/notes.md

```diff
@@ -215,6 +215,15 @@ See the user guide section on the [`plugins {}` block](userguide/plugins.html#se
 ### Example new and noteworthy
 -->
 
+### Experimental caching for C/C++ compilation
+
+This release adds support for using the build cache with C/C++ compilation.
+To enable that feature the system property `org.gradle.native.caching` has to be set to `true`.
+Caching C/C++ compilation is still experimental and some caveats apply:
```

Comment:
I would say:

> This release adds *experimental* support for using the build cache with C/C++ compilation.
To enable that feature the system property `org.gradle.native.caching` has to be set to `true`.
~~Caching C/C++ compilation is still experimental and~~ Some caveats apply:
```